spectrum/test_xpub.py

```diff
@@ -5,7 +5,18 @@
 @pytest.mark.journal
 @pytest.mark.profiles
 @pytest.mark.xpub
-def test_submit_as_anonymous(get_selenium_driver):
+def test_login(get_selenium_driver):
     journal_session = input.JOURNAL.javascript_session(get_selenium_driver())
     xpub_session = journal_session.submit()
     xpub_session.login()
+
+# FUTURE: add to journal_cms when included
+#@pytest.mark.journal_cms
+@pytest.mark.xpub
+def test_initial_submission(get_selenium_driver):
+    journal_session = input.JOURNAL.javascript_session(get_selenium_driver())
```

Comment:
GitHub helpfully syntax highlights, indicating that `input` a builtin function. Might be best to rename (I know it has been there before).

Reply:
Can do that separately